src/pages/Loopstudio/MainSection/MainGallery/MainGallery.jsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import './MainGallery.css';
+import ShowcaseCard from './ShowcaseCard';
+
+function MainGallery() {
+  return (
+    <div className=" mt-20">
+      <h1 className="text-center gallery--text text-2xl uppercase lg:hidden ">
```

Comment:
Same goes here as HeroSection - reuse code. You can move things around with flex item order with the `order` tag. Try to eliminate the duplicating `h1` and `a` elements.

Reply:
Can't use order tag because the buttons and the gallery are not in the same file so flex items order won't work